src/server/src/resources/emails.js

```diff
@@ -25,3 +25,17 @@ export const generateDbConnectionFailedEmail = (err) => {
             <p>${err}</p>`,
   };
 };
+
+export const generateForgotPasswordEmail = (email, name, token) => {
```

Comment:
Add to the Trello a task for Ayelet to write all the emails of the system

Reply:
will do